### DISCUSSION TITLE
SC_VAT_TAX_REPORT format change ???

### DISCUSSION THREAD

**Discussion by mbolder** [amzn/selling-partner-api-models#3685] [General] [answered]
Since he swap to the  sp api, we have requested the SC_VAT_TAX_REPORT successfully and pulled the csv behind the url. 
But since a few days the csv seems to be "encoded" and is not plain text anymore ?

Has there been a change in the response ? If so what format do we get now ?

**Comment by Dragonsangel** [comment, marked as answer]
Just to clarify, are you checking for a value in the `compressionAlgorithm` field of the [getReportDocument](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference#reportdocument) and decoding it with the compression algorithm?

**Comment by mbolder** [comment]
Oh I see, after double checking, I saw that the response is now gzip, we used to get plain csv....... Wonder when they stop playing around .... Thanks for the tip.

**Reply by Dragonsangel** [reply]
No Problem. What I have seen is that compression is applied when the generated report is around 1MB or more.
Might be that your reports have now reached that threshold and hence the compression.
